src/apiCalls.js

```diff
@@ -18,6 +18,15 @@ export const registerNewSessionCallback = (callback: (newTrackingId: string) =>
   _onNewSession = callback;
 };
 
+export const keepAlive = () => {
+  quiqFetch(`${getUrlForContactPoint()}/keep-alive`, {
+    method: 'POST',
+    headers: {
```

Comment:
Nah, it's added right before the call in stubbornFetch to make sure it doesn't get overridden